src/artemis/external_interaction/callbacks/fgs/zocalo_callback.py

```diff
@@ -88,6 +88,34 @@ def wait_for_results(self, fallback_xyz: Point3D) -> Point3D:
                 datacollection_group_id
             )
 
+            # Sort from strongest to weakest in case of multiple crystals
+            raw_results = sorted(
+                raw_results, key=lambda d: d["total_count"], reverse=True
+            )
+            LOGGER.info(f"Zocalo: found {len(raw_results)} crystals.")
+            multi_crystal_msg = (
+                f"Found multiple crystals: {len(raw_results)}."
+                if len(raw_results) > 1
+                else ""
+            )
```

Comment:
Nit: I would just remove this message, it should be obvious from the rest of the comment